modules/friendlytag.js

```diff
@@ -222,7 +222,7 @@ Twinkle.tag.callback = function friendlytagCallback() {
 			]
 		});
 	}
-	form.append({ type: 'submit', className: 'tw-tag-submit' });
+	form.append({ type: 'submit', className: 'tw-tag-submit', name: 'submitButton' });
```

Comment:
Was necessary because by default the button has name `submit`, which causes `form.submit` to refer to this button, shadowing HTMLFormElement API's submit() method.

Reply:
Undone as no longer necessary, submit() is not being used now.